docs/contributing/packages.md

```diff
@@ -1,5 +1,14 @@
-# Packages
+# Package Updates
 
-## Package Updates
+We welcome all contributions that update packages already included in the project Dockerfiles. Follow the process below to update a package version:
 
-## New Packages
\ No newline at end of file
+1. Locate the Dockerfile containing the library you wish to update.
```

Comment:
Is there a way we can make this more specific as with the recipes file?

Reply:
I can extend the description of where the Dockerfiles reside in the repo.